docs/ingestion/tasks.md

```diff
@@ -103,6 +103,14 @@ For some task types, the indexing task can wait for the newly ingested segments
 |`segmentAvailabilityWaitTimeMs`|Milliseconds waited by the ingestion task for the newly ingested segments to be available for query after completing ingestion was completed.|
 |`recordsProcessed`| Partitions that were processed by an ingestion task and includes count of records processed from each partition.|
 
+
+#### Compaction Task Fields
+
+| Field               | Description                                  |
+|---------------------|----------------------------------------------|
+| `segmentsRead`      | # of segments read by a compaction task      |
+| `segmentsPublished` | # of segments published by a compaction task |
```

Comment:
Can you add some commentary here that indicates these fields are only expected for compaction tasks that spawn 1 or more sub tasks